src/SDKs/WebSites/Management.Websites/Microsoft.Azure.Management.Websites.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Websites</PackageId>
 		<Description>Provides website management capabilities for Microsoft Azure.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Websites</AssemblyName>
-		<Version>2.1.0</Version>
+		<Version>2.2.0</Version>
 		<PackageTags>Microsoft Azure website management;website management;</PackageTags>
 		<PackageReleaseNotes><![CDATA[This is the Azure App Service SDK for .NET, based on the Azure App Service REST API, version 2018-11-01 of Certificates, and 2018-02-01 of all other resources]]></PackageReleaseNotes>
```

Comment:
Please update `PackageReleaseNotes` here

Reply:
Updated the notes.